Gemfile

```diff
@@ -8,6 +8,7 @@ gem 'bugsnag', '~> 6.19.0'
 gem 'bundler', '= 2.3.1'
 gem 'faraday', '~> 1.3'
 gem 'github-linguist', '~> 7.13.0'
+gem 'nokogiri', '1.13.1'
```

Comment:
`gem 'nokogiri', '~> 1.13.1'`

Can we use [pessimistic operator](https://thoughtbot.com/blog/rubys-pessimistic-operator), ~> here so that `bundle install` is able to take the patch releases? Patch releases contain security fixes sometimes?